app/org/sagebionetworks/bridge/services/IntentService.java

```diff
@@ -133,6 +140,27 @@ public void registerIntentToParticipate(Study study, StudyParticipant participan
         }
     }
     
+    public boolean registerIntentToParticipate(Study study, Account account) {
```

Comment:
The above registerIntentToParticipate() appears to no longer be used, and this appears to be nearly identical. Instead of creating an overload, would it be better to instead change the old one?

Reply:
This was a mistake on my part, this is mid-refactor. The participant method can go away, removed and altered tests that were testing it to test the other method